src/sdk/pynni/nni/compression/torch/pruners.py

```diff
@@ -199,7 +199,7 @@ def __init__(self, model, config_list):
         k = int(all_bn_weights.shape[0] * config['sparsity'])
         self.global_threshold = torch.topk(all_bn_weights.view(-1), k, largest=False)[0].max()
 
-    def calc_mask(self, layer, config):
+    def calc_mask(self, wrapper):
```

Comment:
we should think about whether expose `wrapper` to `calc_mask` is a good choice

Reply:
It will be nice if users do not need to learn the concept "wrapper", which should be an implementation detail.

Reply:
fixed